optimism/src/keccak/environment.rs

```diff
@@ -148,6 +149,18 @@ pub(crate) trait KeccakEnvironment {
 
     fn length(&self) -> Self::Variable;
 
+    fn two_to_pad(&self) -> Self::Variable;
```

Comment:
For a follow-up: would be nice to document each function, even if it is one-line

Reply:
Opened https://github.com/o1-labs/proof-systems/issues/1616